modulemd/v1/tests/test-modulemd-python.py

```diff
@@ -266,6 +265,36 @@ def test_issue85(self):
         assert mmd3.get_module_components(
         )['testmodule'].peek_ref() == 'private-x'
 
+    def test_issue88(self):
+        # Here, load the same file twice.
+        # All entries in these two lists should be duplicates of each other.
+        objects_from_repo_a = Modulemd.objects_from_file(
+            '%s/test_data/issue88.yaml' % os.getenv('MESON_SOURCE_ROOT'))
+        objects_from_repo_b = Modulemd.objects_from_file(
+            '%s/test_data/issue88.yaml' % os.getenv('MESON_SOURCE_ROOT'))
+
+        # Test at the same priority level
+        prioritizer = Modulemd.Prioritizer()
+        prioritizer.add(objects_from_repo_a, 0)
+        prioritizer.add(objects_from_repo_b, 0)
+        supposedly_merged_objects = prioritizer.resolve()
+
+        # I would expect all three of these to be the same length... but they
+        # are not.
```

Comment:
Maybe adjust this comment and the one below it?

:+1: to including the reproducer almost verbatim as a test.  :)